pro/src/pages/Bookings/__specs__/BookingsRecap.spec.tsx

```diff
@@ -211,7 +211,9 @@ describe('components | BookingsRecap | Pro user', () => {
     await submitFilters()
 
     // When
-    const resetButton = screen.getByText('réinitialiser les filtres')
+    const resetButton = screen.getByText('Réinitialiser les filtres', {
+      selector: '.button-ternary-pink',
```

Comment:
détail : essaye avec juste ça
`const resetButton = screen.getByText('Réinitialiser les filtres')`

si on change la classe ça cassera pas le test !

Reply:
J'ai fait, puis test ne passe pas :  
Comment gérer?

FAIL  pages/Bookings/__specs__/BookingsRecap.spec.tsx > components | BookingsRecap | Pro user > should allow user to reset its pre-filters in the no bookings warning
TestingLibraryElementError: Found multiple elements with the text: Réinitialiser les filtres

Reply:
laisse comme ça, alors il doit y avoir un autre bouton avec le même nom dans la page